src/lib.rs

```diff
@@ -1,13 +1,20 @@
-#[macro_use]
-extern crate diesel;
+#[macro_use] extern crate diesel;
 
 mod types {
     #[allow(deprecated)]
-    use diesel::types::{HasSqlType, NotNull};
+    use diesel::sql_types::{HasSqlType, NotNull};
+    use diesel::SqlType;
     use diesel::pg::{Pg, PgTypeMetadata, PgMetadataLookup};
 
-    #[derive(Clone, Copy)] pub struct TsQuery;
-    #[derive(Clone, Copy)] pub struct TsVector;
+    #[derive(Clone, Copy)]
+    pub struct TsQuery;
+
+    #[derive(Clone, Copy)]
+    pub struct TsVector;
```

Comment:
Can you change that to use `#[derive(SqlType)]` as well? This would fix #19 

(Requires removing the explicit `HasSqlType` and `NotNull` impls below.

Reply:
I've updated, but I confess that I don't know the meaning of exactly what I'm doing =/

Reply:
And @weiznich sorry but I didn't get what you meant in `Looks fine module that semi related requested change there. It would be quite great to fix this minor issue along the way`

Which module/which change/which fix?

Reply:
It's a type, it's meant to be "modulo" instead of "module". So everything beside those small things is fine (+ the version bump for diesel + discussing how we should handle that)